napari/_qt/qt_main_window.py

```diff
@@ -806,6 +813,10 @@ def _update_menu_state(self, menu):
         menu_model = getattr(self, menu)
         menu_model.update_from_context(get_context(layerlist))
 
+    def _update_plugin_menu_state(self):
+        self._update_menu_state('plugins_menu')
```

Comment:
Maybe we should use enum here or annotate `_update_menu_state` with `Literal` to have better readability

Reply:
you mean 
```suggestion
   PLUGIN = 'plugins_menu'
    def _update_plugin_menu_state(self):
        self._update_menu_state(PLUGIN)
```
?

Reply:
ping @Czaki

Reply:
@Czaki can you clarify what you meant here?

Reply:
yes